index.html

```diff
@@ -802,7 +797,7 @@
        starts the domain name lookup, record the time as <a for="PerformanceResourceTiming">domainLookupStart</a>.</li>
       <li>Record the time as <a for="PerformanceResourceTiming">domainLookupEnd</a> immediately after the
        domain name lookup is successfully done. A user agent may need multiple retries before that. If
-                the domain lookup fails, record the time immediately before the user agent aborts the same domain or Timing Allow Origin enabled request for resources, otherwise abort the remaining steps.</li>
+       the domain lookup fails, record the time as domainLookupEnd immediately after the domain name lookup is successfully done, or if the lookup fails and resource passes the <a>timing allow check</a>. A user agent may need multiple retries before that. If the domain lookup fails, set to zero and <a href="#step-final-queue">queue the record</a>.</li>

```

Comment:
"Set to zero and go to step 18" instead of calling queue directly? 

Sorry about the confusion, reviewing the [preview](https://cdn.rawgit.com/wesleyhales/resource-timing/0cbce90b377f5ab8e3c2780000b8dc425e7c7cf2/index.html) once more: 
 - Current steps 1-10 are setup.
 - 11-18 is where all the timestamps are recorded.
  - 12: Record the time as domainLookupEnd immediately after the domain name lookup is successfully done. A user agent may need multiple retries before that. If the domain name lookup fails and resource passes the timing allow check, record the time as domainLookupEnd and go to step 18. 
  - 13-17... ~similar as above
  - 18: If responseEnd end is not set, set it to current time. Record the difference between responseEnd and startTime in duration.
  - ... rest as usual.

With above, I think we can also drop step 7 (in which case offset the step numbers above).

WDYT, does that look more reasonable?